pkg/daemon/update.go

```diff
@@ -286,19 +299,30 @@ func (dn *Daemon) writeFiles(files []ignv2_2types.File) error {
 	return nil
 }
 
+// This is essentially ResolveNodeUidAndGid() from Ignition; XXX should dedupe
```

Comment:
Did https://github.com/openshift/machine-config-operator/pull/49 sneak in? :smile: